presto-spark-base/src/main/java/com/facebook/presto/spark/util/PrestoSparkFailureUtils.java

```diff
@@ -103,8 +104,10 @@ private static List<ExecutionStrategy> getRetryExecutionStrategies(Session sessi
             strategies.add(DISABLE_BROADCAST_JOIN);
         }
 
-        if (isRetryOnOutOfMemoryWithIncreasedMemoryEnabled(session)
-                && (errorCode.equals(EXCEEDED_LOCAL_MEMORY_LIMIT.toErrorCode()) || errorCode.equals(SPARK_EXECUTOR_OOM.toErrorCode()))) {
+        if (isRetryOnOutOfMemoryWithIncreasedMemoryEnabled(session) &&
+                (errorCode.equals(EXCEEDED_LOCAL_MEMORY_LIMIT.toErrorCode())
```

Comment:
Can we default new property with `EXCEEDED_LOCAL_MEMORY_LIMIT, SPARK_EXECUTOR_OOM` and remove this condition